New/src/content/en/events/_index.yaml

```diff
@@ -0,0 +1,71 @@
+project_path: /web/_project.yaml
+book_path: /web/_book.yaml
+landing_page:
+  custom_css_path: /web/events/events.css
+
+  rows:
+
+  - heading: Attend A Developer Event And Get Hacking
```

Comment:
I'd like to see this section removed, and merged with the rest of the content. It feels like a bit of an orphaned heading right now.

Specifically - add a page header:
```
  header:
    description: Attend a developer event and get hacking
```

Then move the rest of the description down into the **Find an event near you**